config/config.example.yml

```diff
@@ -504,6 +504,15 @@ hmac_key: "CHANGE_ME!!"
 ##
 #playlist_length_limit: 500
 
+
+##
+## Email to contact the instance maintainer; used in a mailto: link within the footer. 
+## 
+## Accepted values: Email
+## Default: <none>
+##
+# instance_maintainer_email: 
+
```

Comment:
If we add more customization options (contact e-mail, donate page, etc...) it might be a good thing to group all of that (plus the `banner`) under a dedicated config section, like `customization` or similar.

Reply:
Huh good call! Invidious as of d956b1826e15da6cfcd9a1531b0f1e6ef577dd10 already has two customization options if you also count `modified_source_code_url`. And with this PR, #4007, #3826, and possibly more in the future all adding additional fields, a dedicated customization group will certainly be a good idea.